plugins/MauticSocialBundle/Event/SocialMonitorEvent.php

```diff
@@ -45,6 +45,7 @@ public function __construct($integrationName, Monitoring $monitoring, array $lea
     {
         $this->integrationName  = $integrationName;
         $this->entity           = $monitoring;
+        $this->leadIds          = $leadIds;
```

Comment:
Parameters in docblock are not consistent

Reply:
I'd call it "parameters in docblock add no value". Fixed.